lte/gateway/python/magma/enodebd/data_models/data_model_parameters.py

```diff
@@ -27,7 +27,12 @@ class ParameterName():
     GPS_ENABLE = 'GPS enable'
     GPS_LAT = 'GPS lat'
     GPS_LONG = 'GPS long'
+    GPS_ALTI = 'GPS alti'
     SW_VERSION = 'SW version'
+    VENDOR = "VENDOR"
+    MODEL_NAME = "MODEL name"
```

Comment:
Avoid all caps for the values here.
```suggestion
    VENDOR = "Vendor"
    MODEL_NAME = "Model name"
```